shesmu-pluginapi/src/main/java/ca/on/oicr/gsi/shesmu/plugin/input/Gang.java

```diff
@@ -0,0 +1,20 @@
+package ca.on.oicr.gsi.shesmu.plugin.input;
+
+/** Add a variable to a variable gang */
+public @interface Gang {
```

Comment:
`Add a Shesmu variable to a named group of Shesmu variables`?

From this description it looks like `Gang` refers to a verb (add this to a named group) as opposed to a noun (the named group itself). To me, the verb "gang" conjures images of "gang up on/against", but maybe that's just me? All of which is to say I'd love to sit down with you and talk words.

_pausing review here_

Reply:
Sure. This name came from a discussion with Morgan and Mike where we looked for synonyms for group.

Reply:
Let's just go with it.